salt/modules/file.py

```diff
@@ -4485,6 +4500,10 @@ def check_perms(name, ret, user, group, mode, attrs=None, follow_symlinks=False)
 
     .. versionchanged:: 2014.1.3
         ``follow_symlinks`` option added
+
+    .. versionchanged:: Neon
+
+        Added selinux options
```

Comment:
Same here, this needs to go to the top of the docstring.

Reply:
... and also moved! Blamity!